server/routes/__tests__/postcode.spec.js

```diff
@@ -22,6 +25,10 @@ afterAll(async () => {
 })
 
 describe('postcode page', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
```

Comment:
I don't think this is necessary, and if it is then this is removing the mocking of the flood and address services before each test, meaning that the calls with be run against live services.
I'd like to understand why this is needed

Reply:
I believe this was left in accidentally when I was trying to write the final test I could not get working, I used it to check if any of the other tests / mocks were conflicting with the final one I had written (this may have been the incorrect way to go about this), I then changed the method for how I wrote the test so this should be useless now. I'll go back in and remove this!